<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>20</width>
      <height>181</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>20</width>
      <height>181</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>20</width>
      <height>181</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>20</width>
      <height>181</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="widget0" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>58</width>
      <height>55</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget3" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>58</width>
      <height>55</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget1" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>58</width>
      <height>55</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget2" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>58</width>
      <height>55</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget4" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>58</width>
      <height>55</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget5" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>58</width>
      <height>55</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget6" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>58</width>
      <height>55</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget7" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>58</width>
      <height>55</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget8" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>58</width>
      <height>55</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>279</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionTraining"/>
    <addaction name="actionAI"/>
    <addaction name="Exit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="Exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionTraining">
   <property name="text">
    <string>training</string>
   </property>
  </action>
  <action name="actionAI">
   <property name="text">
    <string>H vs AI</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
